packages/publish-views/lib/shapes.js

```diff
@@ -5,13 +5,23 @@ import path from 'path'
 import { fileURLToPath } from 'url'
 import clownface from 'clownface'
 import { schema, sh } from '@tpluscode/rdf-ns-builders'
+import { createRequire } from 'module'
+import concat from 'barnard59-base/concat.js'
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url))
 const shapePath = path.join(__dirname, '../shapes.ttl')
+const require = createRequire(import.meta.url)
 
 export const viewShape = onetime(async () => {
   const dataset = await $rdf.dataset().import(fromFile(shapePath))
 
   return clownface({ dataset })
     .has(sh.targetClass, schema.Dataset)
 })
+
+export function validationShapes() {
+  const metadataShapes = fromFile(require.resolve('../shapes.ttl'))
```

Comment:
just curious, what's the role of require instead of just, for example, readFile ?

Reply:
In the highlighted line it could have been replaced with `path.resolve(__dirname, '../shapes.ttl'))`. Speaking of, I just notice that both methods are used here so might as well change it.

It's more obvious from the next line, where the argument is a package reference and `require` changes that to a proper filesystem path